frontend/simple/model/contracts/group.js

```diff
@@ -62,10 +62,49 @@ export default DefineContract({
   },
   'GroupPayment': {
     validate: objectOf({
-      payment: string // TODO: change to 'double' and add other fields
+      fromUser: string,
+      toUser: string,
+      date: string,
+      amount: number,
+      currency: string,
+      txid: string,
+      status: string,
+      paymentType: string,
+      details: optional(object),
+      memo: optional(string)
     }),
+    constants: {
+      StatusPending: 'pending',
+      StatusCancelled: 'cancelled',
+      StatusError: 'error',
+      StatusCompleted: 'completed',
+      TypeManual: 'manual',
+      TypeBitcoin: 'bitcoin',
+      TypePayPal: 'paypal'
```

Comment:
Shouldn't we create an Array/List for these Statuses and Payment Types?

Reply:
If you need an array or list out of these constants for some reason it's easy to construct:

```js
Object.keys(contracts.GroupPayment).filter(x => x.indexOf('Type') === 0)
```

Etc.